packages/ui/Scroller/Scroller.js

```diff
@@ -32,9 +32,29 @@ const nop = () => {};
  * @ui
  * @public
  */
+const scrollerDefaultProps = {
```

Comment:
How about moving this definition before the jsdoc?

Reply:
It's moved